spyder/plugins/application/widgets/status.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Status widget for Spyder updates.
+"""
+
+# Standard library imports
+import logging
+
+# Third party imports
+from qtpy.QtCore import Slot
+
+# Local imports
+from spyder.api.widgets.status import StatusBarWidget
```

Comment:
```suggestion
from spyder import get_versions
from spyder.api.widgets.status import StatusBarWidget
```

Move import below to be here so that it's sorted out alphabetically.